### DISCUSSION TITLE
failed to add tag album and no log

### DISCUSSION THREAD

**Discussion by yohcha** [LycheeOrg/Lychee#2740] [Bugs] [answered]
### Lychee version

v6.1.0

### Which PHP version are you using?

PHP 8.2

### Detailed description of the problem

Click "create tag album", but nothing happened:
- no new tag album
- no any log
- no browser f12 report
- no docker log

It's my first time try to create a tag album.
Since it failed without any information, I can't figure out the problem.

### Steps to reproduce the issue

1. Go to example.com/gallery
2. Click "New Tag Album"

### Diagnostics [REQUIRED]

Errors
warning
/v**********************lic/uploads/thumb/c2/e1 has permissions 2770, but should have 2775
warning
/v**********************lic/uploads/small/c2/e1 has permissions 2770, but should have 2775
warning
/v**********************lic/uploads/medium/c2/e1 has permissions 2770, but should have 2775
warning
/v**********************lic/uploads/small2x/c2/e1 has permissions 2770, but should have 2775
warning
/v**********************lic/uploads/thumb2x/c2/e1 has permissions 2770, but should have 2775
warning
1 more directories with wrong permissions
info
Latest version of PHP is 8.3
info
git (software) is not available.
info
Found 925 medium that could be generated.
info
Have you considered supporting Lychee? :)
Info

Lychee Version (tag):                      v6.1.0 (17127bd) -- Up to date (1 hours ago).
DB Version:                                6.1.0
composer install:                          --no-dev
APP_ENV:                                   production
APP_DEBUG:                                 false
APP_URL:                                   set
APP_DIR:                                   default
LOG_VIEWER_ENABLED:                        true
VUEJS_ENABLED:                             true
PHOTO_PIPES:                               true
System:                                    Linux
PHP Version:                               8.2.24
PHP User agent:                            Lychee/4 (https://lycheeorg.github.io/)
Timezone:                                  Asia/Shanghai
Max uploaded file size:                    100M
Max post size:                             100M
Chunk size:                                25.60 MB
Max execution time:                        3600
MySQL Version:                             10.11.10-MariaDB-ubu2204
exec() Available:                          yes
Imagick Available:                         1
Imagick Enabled:                           1
Imagick Version:                           1691
GD Version:                                2.3.3
Number of foreign key:                     12 found.

		

Space
Configuration

version:                                   060100
check_for_updates:                         1
sorting_photos_col:                        title
sorting_photos_order:                      ASC
sorting_albums_col:                        title
sorting_albums_order:                      ASC
imagick:                                   1
skip_duplicates:                           1
small_max_width:                           0
small_max_height:                          360
medium_max_width:                          1920
medium_max_height:                         1080
lang:                                      zh_CN
image_overlay_type:                        desc
default_license:                           none
compression_quality:                       90
grants_full_photo_access:                  1
delete_imported:                           0
mod_frame_enabled:                         1
mod_frame_refresh:                         30
thumb_2x:                                  1
small_2x:                                  1
medium_2x:                                 1
landing_page_enable:                       0
force_migration_in_production:             0
search_public:                             0
hide_version_number:                       0
SL_enable:                                 0
SL_for_admin:                              0
recent_age:                                2
grants_download:                           0
photos_wraparound:                         1
map_display:                               0
zip64:                                     1
map_display_public:                        0
map_provider:                              Wikimedia
force_32bit_ids:                           0
map_include_subalbums:                     0
update_check_every_days:                   3
has_exiftool:                              1
share_button_visible:                      0
import_via_symlink:                        0
has_ffmpeg:                                1
location_decoding:                         0
location_decoding_timeout:                 30
location_show:                             0
location_show_public:                      0
rss_enable:                                0
rss_recent_days:                           7
rss_max_items:                             100
prefer_available_xmp_metadata:             0
editor_enabled:                            1
lossless_optimization:                     0
swipe_tolerance_x:                         150
swipe_tolerance_y:                         250
local_takestamp_video_formats:             .avi|.mov
log_max_num_line:                          1000
unlock_password_photos_with_url_param:     0
nsfw_visible:                              0
nsfw_blur:                                 0
nsfw_warning:                              0
nsfw_warning_admin:                        0
nsfw_banner_override:                      
map_display_direction:                     1
album_subtitle_type:                       oldstyle
upload_processing_limit:                   8
new_photos_notification:                   0
legacy_id_redirection:                     1
zip_deflate_level:                         6
default_album_protection:                  1
album_decoration:                          layers
album_decoration_orientation:              row
allow_username_change:                     1
auto_fix_orientation:                      1
use_job_queues:                            1
random_album_id:                           starred
use_last_modified_date_when_no_exif_date:  0
layout:                                    justified
date_format_photo_thumb:                   M j, Y, g:i:s A e
date_format_photo_overlay:                 M j, Y, g:i:s A e
date_format_sidebar_uploaded:              M j, Y, g:i:s A e
date_format_sidebar_taken_at:              M j, Y, g:i:s A e
date_format_hero_min_max:                  F Y
date_format_hero_created_at:               M j, Y, g:i:s A T
date_format_album_thumb:                   M Y
upload_chunk_size:                         0
nsfw_banner_blur_backdrop:                 0
search_pagination_limit:                   1000
search_minimum_length_required:            2
photo_layout_justified_row_height:         320
photo_layout_masonry_column_width:         300
photo_layout_grid_column_width:            250
photo_layout_square_column_width:          200
photo_layout_gap:                          12
display_thumb_album_overlay:               always
display_thumb_photo_overlay:               hover
default_album_thumb_aspect_ratio:          1/1
use_album_compact_header:                  0
login_button_position:                     left
back_button_enabled:                       1
back_button_text:                          Return to Home
current_job_processing_visible:            1
SA_random_thumbs:                          0
job_processing_queue_visible:              1
login_required:                            0
enable_unsorted:                           1
enable_starred:                            1
enable_recent:                             1
enable_on_this_day:                        1
login_required_root_only:                  1
dark_mode_enabled:                         1
show_keybinding_help_popup:                0
show_keybinding_help_button:               1
disable_se_call_for_actions:               0
enable_se_preview:                         0
slideshow_timeout:                         5
keep_original_untouched:                   1
low_quality_image_placeholder:             0
timeline_photos_enabled:                   1
timeline_photos_public:                    0
timeline_photos_granularity:               day
timeline_photos_order:                     taken_at
timeline_photos_layout:                    square
timeline_photos_pagination_limit:          200
timeline_albums_enabled:                   1
timeline_albums_public:                    0
timeline_albums_granularity:               year
timeline_left_border_enabled:              1
timeline_photo_date_format_year:           Y
timeline_photo_date_format_month:          M Y
timeline_photo_date_format_day:            j M Y
timeline_photo_date_format_hour:           g:i
timeline_album_date_format_year:           Y
timeline_album_date_format_month:          M Y
timeline_album_date_format_day:            j M
hide_nsfw_in_smart_albums:                 0
hide_nsfw_in_frame:                        1
hide_nsfw_in_map:                          1
hide_nsfw_in_search:                       0
search_photos_layout:                      square
hide_nsfw_in_rss:                          1
hide_nsfw_in_timeline:                     0

		

### Browser & System [REQUIRED]

- Browser: Firefox Developer Edition 133.0b9 & Vivaldi 7.0.3495.18 (Stable channel)
- System: Windows 11 Version 24H2 (Build 26100.2314)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment]
> Since it failed without any information, I can't figure out the problem.

~~Can you be sure it is registering the tags?~~ 

Nevermind, tested the dialog box does not open...

**Reply by ildyria** [reply, marked as answer]
Fix incoming: https://github.com/LycheeOrg/Lychee/pull/2743
